python/tchannel/tornado/tornado_req_handler.py

```diff
@@ -0,0 +1,92 @@
+import json
+from tornado import httputil
+from tornado.httputil import RequestStartLine
+from ..req_handler import RequestHandler
+
+
+class TornadoRequestHandler(RequestHandler):
+    def __init__(self, app):
+        self.request_callback = app
+
+    def start_serving(self, request_conn):
+        return _ServerRequestAdapter(self, request_conn)
+
+    def handle_request(self, context, conn):
+        """Handle incoming request
+
+        :param context: incoming message context
+        :param conn: incoming connection
+        """
+        request_delegate = self.start_serving(conn)
+        message = context.message
+        # process http message
+        if message.headers["as"] == "http":
```

Comment:
What's the behavior when `as` is not `http`? You need to have some sort of feedback instead of silently ignoring the error.